### DISCUSSION TITLE
I have two teams(tenant) and total team members are 15 (1st team members 10 and 2nd team members 5) Now how to show only total 2nd team members in navigation bage count.members

### DISCUSSION THREAD

**Discussion by tahinuralam** [filamentphp/filament#12257] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.68

### How can we help you?

 public static function getNavigationBadge(): ?string
    {
        return static::getModel()::count();
    }

**Comment by micraux** [comment, marked as answer]
You can add a `where()` in your request (not tested):
```php
public static function getNavigationBadge(): ?string
{
    return static::getModel()::where('team_id', 2)->count();
}
```

**Reply by tahinuralam** [reply]
Thank you 😄 my dear
It works fine
